client/containers/MainContainer.jsx

```diff
@@ -1,20 +1,15 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-
-const mapStateToProps = (state) => ({});
-
-class MainContainer extends Component {
-  constructor(props) {
-    super(props);
-  }
-
-  render() {
-    return (
-      <div>
-        <h2>Examples</h2>
-      </div>
-    );
-  }
+import React from 'react';
+import ResourceDisplay from '../components/ResourceDisplay';
+import SearchMenu from '../components/SearchMenu';
+import sampleJson from '../../sample-objects.json';
+
+const MainContainer = () => {
+  return (
+    <div>
+      <SearchMenu />
```

Comment:
Are we going to include the search in the menu header, or should this be renamed to just Menubar or something and we'll create a separate search bar? Purely stylistic question, not blocking PR merge.